Gemfile.lock

```diff
@@ -244,6 +244,9 @@ GEM
       actionpack (>= 3.0)
       activesupport (>= 3.0)
       sprockets (>= 2.8, < 4.0)
+    sqlite3 (1.3.10)
```

Comment:
and there's a bug in there preventing us from using in Rails 4. PR submitted... KimNorgaard/validates_hostname#4